### ISSUE TITLE
fix batch_container in sal3 form

### ISSUE THREAD

**Issue by tallenaz** [sul-dlss/libsys-webforms#250]
batch_container needs to return a hash, not an array, and the view should understand that.

`batch_container IN ('NONE', 'MANUSCRIPT_BOX', 'RECORD_BOX', ‘OTHER_BOX’, 'UNKNOWN' )`

so, for example:

```
{ manuscript_box: 'MANUSCRIPT_BOX', etc }
```

**Comment by jgreben** [comment]
This is what it will return after making the change:
```
<select size="4" class="form-control" name="sal3_batch_requests_batch[batch_container]" id="sal3_batch_requests_batch_batch_container">
  <option value="MANUSCRIPT_BOX">Manuscript Box</option>
  <option value="RECORD_BOX">Record Storage Boxes</option>
  <option value="OTHER_BOX">Other Box Type</option>
  <option value="UNKNOWN">Unknown</option>
</select>
```

**Comment by jgreben** [comment]
Closed by #271
